0021-zarf-values/README.md

```diff
@@ -0,0 +1,306 @@
+
+### Goals
+
+- Design a replacement for Zarf Variables for members of the community familiar with Helm
+- Provide more flexibility for what values can be and how they can be used
+- Integrate this new design to work across the features where Zarf Variables work today
+
+### Non-Goals
+
+- Entirely remove Zarf Variables, Constants and Templates
+  - (this may be desireable once the new design is solidified, but Zarf Variables enable some functionality that we may want to preserve (since they are template-driven))
+
+## Proposal
+
+The proposed solution is to add a new `values` global field to the Zarf package configuration that will accept a list of values files to serve as package defaults as well as an optional schema file for validating the values provided.  These fields would follow existing Zarf compose conventions and would also map into Helm charts with a new `values` field under `charts`. The Zarf configuration itself would also change to allow Go templating of values in Zarf actions instead of being injected into the environment like Zarf Variables are today.
+
+To set these values new `package.[deploy|remove].values` configuration options would be added to the Viper config and a new `-f`/`--values` flag would be added to the CLI to allow values files to be specified on `zarf package deploy`, `zarf package remove` and `zarf dev deploy`.  For now, the `--set` flag would remain as it is for Zarf Variables though eventually we may want to deprecate it and align to the [Helm `--set` syntax](https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set) with the values specified setting Zarf Values instead of Zarf Variables.
+
+### User Stories (Optional)
+
+#### Story 1
+
+**As** Jacquline **I want** to be able to specify values files for a Zarf package **so that** I can simplify package configurations and rely on my existing familiarity with Helm.
+
+**Given** I have a Zarf Package with a Helm value override in a chart
+```yaml
+components:
+  - name: my-component
+    charts:
+      - name: mychart
+        version: 0.1.0
+        namespace: zarf
+        localPath: chart
+        valuesFiles:
+          - values.yaml
+        values:
+          # option 1: 
+          - sourcePath: my-component.resources
+            targetPath: resources
```

Comment:
I like option1 the best. Clarity > brevity imo

Reply:
I tend to agree though would like others' thoughts too

Reply:
I would vote for option 1 as well. My question then becomes - what is the UX for wanting to include the entirety of a key in the `values.yaml` file? 

```yaml
- sourcePath: my-component
            targetPath: ?
```